undertow/WFLY-12473_UUID-instance-id.adoc

```diff
@@ -0,0 +1,65 @@
+= WFLY-12473 UUID instace-id
+:author:            Flavia Rainone
+:email:             frainone@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          instance-id,uuid,uuid-instance-id
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This feature is to be able to set the value of UndertowService.instanceId to a UUID,
+if instance-id was not defined and user desires to have it so.
+EAP6 Web Server would set it as a default whenever instance-id configuration was absent, and Undertow defaults to ${jboss.node.name} instead.
+Some customers would like to have it default to a UUID instead.
```

Comment:
@fl4via why adding the new attribute `uuid-instance-id` instead of setting `instance-id` to an expression like `${UUID}` instead of `${jboss.node.name}` ? I am asking because on WFLY-12473 I read `Since we can't change it for compatibility reasons a new attribute such as 'uuid-instance-id' can be added; if it was true then it would force the instance id to a UUID.` and I can't see why we can't set it explicitly instead of introducing a new attribute given the final effect is `instance-id` isn't `undefined` any more